components/FullwidthSlider/src/index.scss

```diff
@@ -1,125 +1,127 @@
-  position: var(--denhaag-fullwidth-slider-wrapper-position);
+  position: relative;
 }
 
 .denhaag-fullwidth-slider__slide {
-  width: var(--denhaag-fullwidth-slider-slide-width);
+  width: var(--denhaag-fullwidth-slider-slide-width, 100%);
 }
 
 .denhaag-fullwidth-slider__card {
-  position: var(--denhaag-fullwidth-slider-card-position);
-  padding-inline-start: var(--denhaag-fullwidth-slider-card-padding);
-  padding-inline-end: var(--denhaag-fullwidth-slider-card-padding);
-  padding-block-start: var(--denhaag-fullwidth-slider-card-padding);
-  padding-block-end: var(--denhaag-fullwidth-slider-card-padding);
-  display: var(--denhaag-fullwidth-slider-card-display);
-  justify-content: var(--denhaag-fullwidth-slider-card-justify-content);
-  border-bottom: var(--denhaag-fullwidth-slider-card-border-bottom);
-  border-color: var(--denhaag-fullwidth-slider-card-border-color);
-  opacity: var(--denhaag-fullwidth-slider-card-opacity);
-  pointer-events: var(--denhaag-fullwidth-slider-card-pointer-events);
-  background-color: var(--denhaag-fullwidth-slider-card-background-color);
-  transition: var(--denhaag-fullwidth-slider-card-transition);
-  transform: var(--denhaag-fullwidth-slider-card-transform);
-  margin-block-start: var(--denhaag-fullwidth-slider-card-margin-block-start-xs);
-  margin-inline: var(--denhaag-fullwidth-slider-card-margin-inline-xs);
-  width: var(--denhaag-fullwidth-slider-card-width-xs);
+  position: relative;
+  padding-inline: var(--denhaag-fullwidth-slider-card-padding, 16px);
+  padding-block: var(--denhaag-fullwidth-slider-card-padding, 16px);
+  display: flex;
+  justify-content: space-between;
+  border-bottom: var(--denhaag-fullwidth-slider-card-border-bottom, 1px solid);
+  border-color: var(--denhaag-fullwidth-slider-card-border-color, var(--denhaag-color-grey-2));
+  opacity: var(--denhaag-fullwidth-slider-card-opacity, 0);
+  pointer-events: var(--denhaag-fullwidth-slider-card-pointer-events, none);
+  background-color: var(--denhaag-fullwidth-slider-card-background-color, var(--denhaag-color-white));
+  transition: var(--denhaag-fullwidth-slider-card-transition, all 500ms ease-in-out);
+  transform: var(--denhaag-fullwidth-slider-card-transform, translateX(12px));
+  margin-block-start: var(--denhaag-fullwidth-slider-card-margin-block-start, -32px);
+  margin-inline: var(--denhaag-fullwidth-slider-card-margin-inline, 24px);
+  width: var(--denhaag-fullwidth-slider-card-width, auto);
 
   @media (prefers-reduced-motion) {
-    transition-delay: var(--denhaag-fullwidth-slider-slide-active-card-prefers-reduced-motion-transition-delay);
-    transition-duration: var(--denhaag-fullwidth-slider-slide-active-card-prefers-reduced-motion-transition-duration);
+    transition-delay: var(--denhaag-fullwidth-slider-slide-active-card-prefers-reduced-motion-transition-delay, 0ms);
+    transition-duration: var(
+      --denhaag-fullwidth-slider-slide-active-card-prefers-reduced-motion-transition-duration,
+      0,
```

Comment:
Mag dit? Voglensmij mag de `var()`maar 1 fallback waarden hebben.

Reply:
Ik twijfelde ook, maar Claude zei me dat alles achter de eerste comma de fallback value is :D